README.md

```diff
@@ -1016,9 +1016,13 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 
 ## Package Management
 
-*Libraries for package and dependency management.*
+*Official tooling for package management*
 
 * [dep](https://github.com/golang/dep) - Go dependency tool.
+* [vgo](https://go.googlesource.com/vgo/) - Versioned Go.
```

Comment:
that's 404 :)

Reply:
Interesting, it does not 404 for me, see below
![image](https://user-images.githubusercontent.com/10098811/44096987-9b44ffe4-9fd4-11e8-8e0c-04d9ca5620a8.png)

Reply:
Nor me!

Reply:
oh, never mind, got fixed for me